docs/resiliency-checklist.md

```diff
@@ -17,71 +17,97 @@ description: How to design your application to have as much uptime as possible.
+Our platform is extremely highly available, (99.99% available as of mid-2020!) but that doesn't mean you should assume that your application is guaranteed the same - application outages can happen for reasons other than full platform outages.
 
-Monitoring can be performed through health checks - these are scripts which check for a certain outcome on your pod, and can alert you if the expected outcome doesn't occur.
-They can be a great first indicator of problems, and can also cause the pod to restart on its own, which may be enough to fix the issue without requiring your intervention.
+The best way to keep on top of these issues in a proactive manner is to monitor your application and ensure that there are appropriate notifications of issues. There are tons of ways that such a monitoring can be implemented:
+* pod health checks
+* uptime dashboards
+* API health checks
+* storage health checks
+* and more!
 
-There are other options for monitoring as well, but these often vary depending on the specific nature of the project/application in question.
+Many of these notification options provide your team an opportunity to act to prevent an outage before one even starts - finding out, for example, that your storage is almost full _before_ it fills up means that your team can act to deal with the storage issue _before_ it causes an outage or service issue.
+Still others let you know the moment an outage starts and can provide extremely valuable information about the cause of the outage, so your team can begin troubleshooting right away without having to wait for users to inform you of the problem.
 
-### Set up your application to be Highly Available!
+### A Highly Available App
 
-You should have multiple accessible pods running your application at once - **three is the ideal minimum**.
-If any single pod fails, you will still have two other pods in operation while your broken pod recovers.
-In the case of most applications, this is relatively straight-forward and is covered in our [Openshift 101](https://developer.gov.bc.ca/ExchangeLab-Course:-Openshift-101) course.
-It can be a little more complicated to set up a highly available database, depending on your database of choice.
-If you're looking to use Postgres, check out [Patroni](https://github.com/BCDevOps/platform-services/tree/master/apps/pgsql/patroni) - an open-source, highly-available version of Postgres for use on containerized platforms.
+Our platform may have _very_ high availability in general, but our individual _nodes_ do not. This doesn't mean the nodes are unstable or difficult to use - it just means that the way we approach maintenance and infrastructure problems are a little different from the way things work in the legacy application world.
+Did you know that we don't guarantee a single node will be up for more than 24 hours at a time? That's right - our nodes *can* be restarted or changed very often!
```

Comment:
So I guess my thought on getting super prescriptive is that it encourages a certain element of dependence on us - teams come to the Platform Team to learn how to architect their applications, which really should be on them. It's way too much to say that we should be, in addition to actually running the platform, the primary experts in application architecture too.

Reply:
And I don't think it's true that the documentation of specific solutions doesn't fit the document at all anymore - it's just that such documentation would get moved to the readme of a project that actually uses the QoS pods (for example), rather than being here. This doc becomes more a hub of places for _other people_ to be like "here are the specific resilience methods we use in our project." And there's nothing at all that says _we_ can't add examples of our own platform service projects into that mix.